content/community/resources/howtocontribute.md

```diff
@@ -3,13 +3,11 @@ title: "Apache Hive : HowToContribute"
 date: 2024-12-12
 ---
 
-# Apache Hive : How to Contribute
-
 This page describes the mechanics of *how* to contribute software to Apache Hive. For ideas about *what* you might contribute, please see open tickets in [Jira](https://issues.apache.org/jira/browse/HIVE).
 
 {{< toc >}}
```

Comment:
Off-topic: We may discuss the guidelines about how to headline or if we should remove the toc for all contents.